code/obj/item/clothing/helmets.dm

```diff
@@ -50,6 +50,7 @@
 	see_face = FALSE
 	item_state = "s_helmet"
 	var/on = 0
+	var/use_alt_art = FALSE
```

Comment:
Standard approach here would be to have `var/base_icon_state` which the flashlight toggle would set the real iconstate based off of

Reply:
Had to shift the sprite names to get them in-line with a base icon state format. Thank you!